### PR TITLE
Update infos on how to apply patches or upgrade

### PR DESCRIPTION
**Pull request by LaureBro**
**Description**

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: migrate_pim/index.rst [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
 ===========================
 
-This chapter describes how to upgrade from a patch or minor version to a more recent one.
+This chapter describes how to apply a patch on your current version or how to upgrade from a minor/major version to a more recent one.
 
 As a prerequisite, we assume that you are using a project installed from a Community Standard (:doc:`/install_pim/index`).
 
-Patch Version
+How to apply a patch?
 -------------
```
Comment (damien-carcel) [inline migrate_pim/index.rst]: must be the same length than its sentence

File: migrate_pim/index.rst [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
+Also for support purposes, if you experience an issue, make sure your PIM is up-to-date before raising a ticket to our Support team.
+ 
+When our maintenance team takes care of issues, they take extreme care to avoid any backward compatibility break, but it might happen. If it is the case, the BC break is mentioned in the version's changelog. 
+ 
+Akeneo PIM CE changelog is available on our GitHub repository: 
 
-When fixing these issues, we take extreme care to avoid any backward compatibility break.
+* Go to our `Community Repository <https://github.com/akeneo/pim-community-dev/>`_. 
+* Select your branch in the drop down menu, for instance select 2.3 branch.
+* Then scroll down the page, and select the desire Changelog in the list, for instance `Changelog 2.3 version <https://github.com/akeneo/pim-community-dev/blob/2.3/CHANGELOG-2.3.md>`_.
```
Comment (damien-carcel) [inline migrate_pim/index.rst]: select the desire**d**

File: migrate_pim/index.rst [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
+---------------------
 
-We frequently release patch versions, in order to provides security and bug fixes.
+We frequently release patches for all supported versions - you can find the list of supported versions in this article 
+`How many PIM Enterprise Edition versions are maintained by Akeneo? <https://www.akeneo.com/fr/knowledge-base/how-many-pim-versions-are-maintained-by-akeneo/>`_      
+ 
+Each patch contains bug fixes and sometimes improvements or security fixes, **that is why it is very important to always have the latest patch installed on your PIM.** 
+Also for support purposes, if you experience an issue, make sure your PIM is up-to-date before raising a ticket to our Support team.
+ 
+When our maintenance team takes care of issues, they take extreme care to avoid any backward compatibility break, but it might happen. If it is the case, the BC break is mentioned in the version's changelog.
```
Comment (phaseinducer) [inline migrate_pim/index.rst]: "fixes issues" ? (to avoid the repetition)

Comment (phaseinducer) [reply migrate_pim/index.rst]: changelog of the version

File: migrate_pim/index.rst [hunk 4]
```diff
@@ -163,10 +177,10 @@ Then, follow the migration guides located in your archive to upgrade your projec
 * From EE v1.0 to EE v1.3: UPGRADE-1.3.md
 
 
-Major Version
--------------
+How to upgrade to a major version?
+----------------------------------
 
-We release major versions every year, these versions bring new features and larger changes to answer to growing needs.
+We release a major version each year, these new major version brings new features and larger changes to answer to clients growing needs.
```
Comment (phaseinducer) [inline migrate_pim/index.rst]: "to answer clients growing needs"

File: migrate_pim/index.rst [hunk 5]
```diff
@@ -163,10 +177,10 @@ Then, follow the migration guides located in your archive to upgrade your projec
 * From EE v1.0 to EE v1.3: UPGRADE-1.3.md
 
 
-Major Version
--------------
+How to upgrade to a major version?
+----------------------------------
 
-We release major versions every year, these versions bring new features and larger changes to answer to growing needs.
+We release a major version each year, these new major version brings new features and larger changes to answer clients growing needs.
```
Comment (damien-carcel) [inline migrate_pim/index.rst]: **this** new major version

Comment (damien-carcel) [reply migrate_pim/index.rst]: or these new major versions bring

Comment (LaureBro) [reply migrate_pim/index.rst]: went for "these new major versions bring" thanks!

**Pull request merged**